packages/graphql/tests/integration/directives/fulltext.int.test.ts

```diff
@@ -74,672 +95,2324 @@ describe("@fulltext directive", () => {
+            expect((gqlResult.data?.[queryType] as any[])[0][personTypeLowerFirst]).toEqual({
+                name: person3.name,
+            });
+            expect((gqlResult.data?.[queryType] as any[])[1][personTypeLowerFirst]).toEqual({
+                name: person1.name,
+            });
+            expect((gqlResult.data?.[queryType] as any[])[2][personTypeLowerFirst]).toEqual({
+                name: person2.name,
+            });
+            expect((gqlResult.data?.[queryType] as any[])[0][SCORE_FIELD]).toBeGreaterThanOrEqual(
+                (gqlResult.data?.[queryType] as any[])[1][SCORE_FIELD]
+            );
+            expect((gqlResult.data?.[queryType] as any[])[1][SCORE_FIELD]).toBeGreaterThanOrEqual(
+                (gqlResult.data?.[queryType] as any[])[2][SCORE_FIELD]
+            );
+        });
+        test("Score isn't returned if not requested", async () => {
+            const query = `
+                query {
+                    ${queryType}(phrase: "some name") {
+                        ${personTypeLowerFirst} {
+                            name
+                        } 
+                    }
+                }
+            `;
+            const gqlResult = await graphql({
+                schema: generatedSchema,
+                source: query,
+                contextValue: {
+                    driver,
+                    driverConfig: { database: databaseName },
+                },
+            });
+
+            expect(gqlResult.errors).toBeFalsy();
+            expect(gqlResult.data?.[queryType]).toEqual([
+                {
+                    [personTypeLowerFirst]: {
+                        name: person3.name,
+                    },
+                },
+                {
+                    [personTypeLowerFirst]: {
+                        name: person1.name,
+                    },
+                },
+                {
+                    [personTypeLowerFirst]: {
+                        name: person2.name,
+                    },
+                },
+            ]);
+        });
+        test("Nodes aren't returned if not requested", async () => {
+            const query = `
+                query {
+                    ${queryType}(phrase: "some name") {
+                        score
+                    }
+                }
+            `;
+            const gqlResult = await graphql({
+                schema: generatedSchema,
+                source: query,
+                contextValue: {
+                    driver,
+                    driverConfig: { database: databaseName },
+                },
+            });
+
+            expect(gqlResult.errors).toBeFalsy();
+            expect((gqlResult.data?.[queryType] as any[])[0][personTypeLowerFirst]).toBeUndefined();
+            expect((gqlResult.data?.[queryType] as any[])[1][personTypeLowerFirst]).toBeUndefined();
+            expect((gqlResult.data?.[queryType] as any[])[2][personTypeLowerFirst]).toBeUndefined();
```

Comment:
It may be easier to directly test the whole data array, instead of going one by one.

Using something like `toIncludeSameMembers`

Reply:
Also the order does matter for these tests as they are supposed to be ordered by score DESC by default so it's safe to assert these individually